website/docs/reference/resource-configs/persist_docs.md

```diff
@@ -120,11 +120,49 @@ resources as needed.
 
 ## Support
 
-The `persist_docs` config is supported on all core dbt plugins: BigQuery,
-Redshift, Snowflake, and Postgres. Some databases impose limitations on the
-types of descriptions that can be added to database objects. At present, the
-`persist_docs` flag has the following known limitations:
- - Column-level comments are not supported on Snowflake <Term id="view">views</Term>
+The `persist_docs` config is supported on the most widely used dbt adapters:
+- Postgres
+- Redshift
+- Snowflake
+- BigQuery
+- Apache Spark & Databricks
+
+However, some database limit where and how descriptions can be added to database objects. Those database adapters may not support `persist_docs`, or may offer only partial support.
```

Comment:
```suggestion
However, some databases limit where and how descriptions can be added to database objects. Those database adapters might not support `persist_docs`, or might offer only partial support.
```

Reply:
suggestion: use `might` instead of `may`. 

motivation: `May` can also mean to have permission like, "you may have your drink now", so `might` is preferred to help make translation/localization easier.